### ISSUE TITLE
Promoting a user by sending an email is not working on our custom version of managed users

### ISSUE THREAD

**Issue by valentinchaput** [OpenSourcePolitics/decidim#294]
**Describe the bug**
On Angers + Demo, we face an error alert when filling the address of a managed user we want to promote.
<img width="1277" alt="capture d ecran 2018-10-02 a 08 30 12" src="https://user-images.githubusercontent.com/10398564/46332950-b3a42e00-c61e-11e8-80d9-05eda36964a4.png">
It is related to the last update as the process was working last week (before the update regarding new budget translations) and it is still working on other platforms (see below). 

**To Reproduce**
As an admin (mako), go to https://pompano.osp.cat/ > Users > "Utilisateurs représentés" > Add a new one > do something on the platform > close the session & back to the admin panel > try to promote this user with an email. 

**Expected behavior**
Same as Nancy, whose platform is still working this morning:
<img width="1276" alt="capture d ecran 2018-10-02 a 08 37 23" src="https://user-images.githubusercontent.com/10398564/46332902-85265300-c61e-11e8-83e5-647826c39eaf.png">
<img width="952" alt="capture d ecran 2018-10-02 a 08 37 35" src="https://user-images.githubusercontent.com/10398564/46332907-86f01680-c61e-11e8-8786-b55a69c017e2.png">

This is pretty urgent...

**Comment by moustachu** [comment]
@lu-hamani @valentinchaput 
Apparently, the exact bug is that the error message are not displayed correctly in the front end (admin) (and the console / log) : 

**The user cannot be promoted because the email address is already in use for another user**

We need to decide if we patch this as client like Orsay might be disoriented by this behavior.

**Comment by lu-hamani** [comment]
@valentinchaput @moustachu
I tried to replicate the bug on my own on pompano and on demo as @valentin did. I actually couldn't and I managed to represent a user as expected. On the other hand, I encountered another issue (on pompano only, demo works fine), since when I promote a UR, he doesn't receive the invitation email. 

I also tested locally the workflow (on the 0.12 stable-budget branch) and it works fine. I get my invites email in letter_opener.

Also, in the mail thread with the client she insists that the issue is not restricted to using the same email for 2 different users.  
So to sum it up, @moustachu and @armandfardeau, do you have any suggestions on how I may track and define the bug for you before either one of you has to work on it ?

**Comment by lu-hamani** [comment]
The issue is actually linked to the invitation email following the promotion of an IR.

User is promoted successfully but doesn't receive the invitation email. Yet he receives all other email notifications ("confirmation email" for instance). The bug is present on https://pompano.osp.cat/ 

Users not receiving emails use different email services (provided by the city, gmail, yahoo).

**Comment by moustachu** [comment]
Okay ! So the `%{user_name}` used in the text for the invitation email were buggy. 

Fixed on 0.12-stable --> 0.12-stable-budget --> osp-app#alt/angers

@lu-hamani Should be OK on https://pompano.osp.cat/ (Delivered & Tested ✅ )

**Comment by lu-hamani** [comment]
@moustachu 
this one has come back for Angers. This is because of it that you add to "manually" send invites last week.
